### PR TITLE
Updating documentation

### PR DESCRIPTION
**Pull request by leovms**

### CODE REVIEW

File: Artifacts/AUTHORING.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Authoring Artifacts
+
+An Artifact consists of an artifact definition file and other script files that are stored
+in a folder in a git repository. Artifact definition file consists of JSON and expressions
+which you can use to specify what you want to install on a machine. For example, you can 
+define the name of artifact, command to run and parameters that are made available when
```
Comment (joerage) [inline Artifacts/AUTHORING.md]: the name of the artifact

File: Artifacts/AUTHORING.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      }
+    }
+
+
+|Element Name |Required |Description
+|-------------|:--------|:-----------
+|$schema      |No       |Location of the JSON schema file that helps in testing the validity of the definition file.
+|title        |Yes      |Name of the Artifact that is displayed in Lab
+|description  |Yes      |Description of the Artifact that is displayed in Lab
+|iconUri      |No       |Uri of the icon that will be displayed in the Lab
```
Comment (joerage) [inline Artifacts/AUTHORING.md]: remove 'the' to be consistent.

File: Artifacts/AUTHORING.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+          "displayName": "<display-name-of-parameter>",
+          "description": "<description-of-parameter>"
+        }
+      }
+
+|Element Name |Required |Description
+|:------------|:--------|:----------
+|type         |Yes      |Type of parameter value. See the list below of the allowed types.
+|displayName  |Yes      |Name of the parameter that is displayed to a user in Lab
+|description  |Yes      |Description of the parameter that is displayed to a used in Lab
```
Comment (joerage) [inline Artifacts/AUTHORING.md]: Review description of the decription element.

File: Artifacts/AUTHORING.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+The allowed types are:
+- string – any valid JSON string 
+- int – any valid JSON integer
+- bool – any valid JSON Boolean
+- array – any valid JSON array
+
+### __Expression and Functions__
+
+You can use expression and functions to construct the artifact install command.
```
Comment (joerage) [inline Artifacts/AUTHORING.md]: `an expression` or `expressions`

File: Artifacts/AUTHORING.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+-------------------------------------------------------------------------------
+
+## Getting Started
+
+Create your custom artifact by following steps below:
+- Install a JSON editor
+
+  You will need a JSON editor to work on artifact definition file. We recommend using 
+  Visual Studio Code which you can install from here. Visual Studio Code is available
```
Comment (joerage) [inline Artifacts/AUTHORING.md]: Add link

File: Artifacts/README.md [hunk 6]
```diff
@@ -1,144 +1,20 @@
...
-     ![Artifact-folder]( .../../../Documentation/images/artifact-folder-github.png)
- 
-
-- Add an Artifacts Repository in Lab
-
-  Please [check out instructions here](https://azure.microsoft.com/en-us/documentation/articles/devtest-lab-add-artifact-repo/) to add an Artifacts Repository in Lab
+Create a pull request that includes
+* a detailed description of the new contribution or the change,
+* steps on how to test the artifact and/or changes, and
+* screenshots of your the expected output, where possible, from your own testing.
```
Comment (joerage) [inline Artifacts/README.md]: `screenshots of the expected output`

File: Environments/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+Contributions to the Public Environment Repo are limited to corrections in the existing code base. At times, new environment will be welcome. However, the team reserves the right to decide which contributions will be accepted.
+
+The process to get changes published requires the team to review the code submissions and test the environment contributions or the changes. Submit a pull request to get the process started.
+
+### Submission Requirements
+
+Create a pull request that includes
+* a detailed description of the new contribution or the change,
+* steps on how to test the environment and/or changes,
+* screenshots of your the expected output, where possible, from your own testing, and
```
Comment (joerage) [inline Environments/README.md]: `screenshots of the exected ouput`

File: samples/ClassroomLabs/SdkSamples/README.md [hunk 8]
```diff
@@ -1,16 +1,16 @@
...
+  * Set the `Sign-on URL` to https://dtlclient. This doesn't matter at this time.
+* Go to the registered app page for your new app.
+  * Copy the "ApplicationID" to your notes. It will be used as the `ClientId` in the `app.config`.
+  * Go to "Settings/Owners" and add both your account as an owner.
+  * Go to "Settings/Redirect URIs" and make sure "https://dtlclient" is on there.
+  * Go to "Settings/Required Permissions" and click "Add". Type in "Windows Azure Service Management API" and check all permissions boxes.
+  * On "Settings/Required Permissions" click "Grant Permissions". This should succeed.
+* Next up, we can actually go create a lab account to make SDK calls against. Go to the Lab Services blade and create a new LabAccount.
+* Once that LabAccount has been created, go to "Access Control (IAM)" and add your account as an owner of the lab account.
+* Now you can make set the rest of the options in the `app.config` and start making SDK calls!
```
Comment (joerage) [inline samples/ClassroomLabs/SdkSamples/README.md]: `Now you can set`

File: samples/DevTestLabs/QuickStartTemplates/101-dtl-create-vm-username-pwd-customimage-with-expiration/README.md [hunk 9]
```diff
@@ -1,14 +1,13 @@
...
 
-
 This deployment template is generally used for creating a virtual machine using a custom image (pointing to a sysprepped VHD file) .
 
 This template creates a new virtual machine in a Lab with a expiration date.
 - A new user account is created using the username/password combination on the virtual machine.
 - The user account is added to the local administrators group.
 - A expiration date is set for the VM.
-- VM will be deleted within 24 hrs of expiration date.
+- VM will be deleted within 24 hrs. of expiration date.
```
Comment (joerage) [inline samples/DevTestLabs/QuickStartTemplates/101-dtl-create-vm-username-pwd-customimage-with-expiration/README.md]: Remove dot. Replace `hrs` with `hours`

File: samples/README.md [hunk 10]
```diff
@@ -0,0 +1,3 @@
+# Azure Lab Services Samples
+
+This folder contains sample ARM Templates, Scripts and SDK projects to help demonstrate integration with Classroom Labs and DevTest Labs.
```
Comment (joerage) [inline samples/README.md]: Small `s` in `scripts`

File: tasks/ReadMe.md [hunk 11]
```diff
@@ -2,11 +2,23 @@
...
-# References
+## Contributions
+
+Contributions are encouraged and welcome. However, the team reserves the right to decide which contributions will be included in future releases of the extension.
+
+The process to get changes published requires the team to review, test and package the extension. Submit a pull request to get the process started.
+
+### Submission Requirements
+
+Create a pull request that includes the nature of the change, steps on how to test it and screenshots of your the expected output, where possible, from your own testing.
```
Comment (joerage) [inline tasks/ReadMe.md]: `of your expected output`

File: Artifacts/AUTHORING.md (hunk 2 above)
Comment (leovms) [reply Artifacts/AUTHORING.md]: Instead, I added `the` where needed to make it read better.

**Pull request merged**